src/test/javascript/spec/component/programming-exercise/programming-exercise-configure-grading.component.spec.ts

```diff
@@ -362,6 +362,54 @@ describe('ProgrammingExerciseConfigureGradingComponent', () => {
         flush();
     }));
 
+    it('should show error alert when test case weights are less or equal zero', fakeAsync(() => {
```

Comment:
I do not think you need fakeAsync here. I removed both this and the ticks and it still works

Reply:
Done in `8177c86`